src/components/ui/fixed-toolbar-buttons.tsx

```
@@ -0,0 +1,71 @@
+
+import { MarkToolbarButton } from './mark-toolbar-button';
+import { ToolbarGroup, ToolbarSeparator } from './toolbar';
+import { FontFamilyToolbarButton } from './font-family-toolbar-button';
+
+export function FixedToolbarButtons() {
+  return (
+    <>
+      <ToolbarGroup>
+        <FontFamilyToolbarButton />
+      </ToolbarGroup>
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)